build.go

```diff
@@ -166,6 +166,7 @@ var targets = map[string]target{
 			{src: "man/strelaysrv.1", dst: "deb/usr/share/man/man1/strelaysrv.1", perm: 0644},
 			{src: "cmd/strelaysrv/etc/linux-default/syncthing-relaysrv", dst: "deb/etc/default/syncthing-relaysrv", perm: 0644},
 			{src: "cmd/strelaysrv/etc/linux-systemd/system/strelaysrv.service", dst: "deb/lib/systemd/system/strelaysrv.service", perm: 0644},
+			{src: "etc/firewall-ufw/strelaysrv", dst: "deb/etc/ufw/applications.d/strelaysrv", perm: 0644},
```

Comment:
Is there a dir in `/usr` for that?
Because normally `/etc` should be user-configuration

Reply:
I'm not 100% sure, but I'd say no, because standard Ubuntu packages deploy files in `/etc/ufw/applications.d/`
See for example for apache: https://packages.ubuntu.com/xenial/amd64/apache2/filelist

Also we already put the syncthing UFW file here, see https://github.com/syncthing/syncthing/blob/8ebd3834dbe85019a1faa9705933cbae583390e1/build.go#L113